Basic-Captions/js/app.js

```diff
@@ -0,0 +1,141 @@
+/* global $ OT SAMPLE_SERVER_BASE_URL */
+
+var apiKey;
```

Comment:
super nit picky, buy given that this is customer facing shouldn't we use `const` & `let`?

Reply:
good point, I just copy-pasted the skeleton from a different sample app. will update 🚀